Implicit any type error, when compiling with strict set to true in tsconfig.

# Bug Report

#### What is the expected behavior?

Clean compilation with strict set to true in tsconfig.

#### What is the current behavior?

ERROR in ../../node_modules/@angular/flex-layout/core/typings/media-trigger/media-trigger.d.ts:66:17 - error TS7033: Property 'currentActivations' implicitly has type 'any', because its get accessor lacks a return type annotation.

#### What are the steps to reproduce?

Add  "@angular/flex-layout": "9.0.0-beta.29" to your package.json
Compile your project using the strict flag set to true in tsconfig.json

#### What is the use-case or motivation for changing an existing behavior?

Angular flex layout should not be a limiting factor in being able to use strict typescript complication options.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?

Angular 9.0.5
TypeScript 3.6.4

--- a/src/lib/core/media-trigger/media-trigger.ts
+++ b/src/lib/core/media-trigger/media-trigger.ts
@@ -192,7 +192,7 @@ export class MediaTrigger {
     this.matchMedia.source.next(new MediaChange(matches, query));
   }
 
-  private get currentActivations() {
+  private get currentActivations(): string[] {
     return this.matchMedia.activations;
   }
 

